include/crocoddyl/core/utils/deprecate.hpp

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2020, University of Edinburgh
+// Copyright (C) 2020, University of Edinburgh, CNRS, INRIA
```

Comment:
Is OK to use CNRS?
I mean we always use LAAS-CNRS.

Reply:
CNRS is the real holder of copyright here. LAAS is not mandatory, but not an issue. If you prefer to uniformize, I can change it here.

Reply:
No, I don't have any issue.